velox/functions/prestosql/InPredicate.cpp

```diff
@@ -253,7 +253,7 @@ class InPredicate : public exec::VectorFunction {
     // Indicates whether result can be true or null only, e.g. no false results.
     const bool passOrNull = filter_->testNull();
 
-    if (arg->isConstant(rows)) {
+    if (arg->isConstantEncoding()) {
```

Comment:
So the only semantic change is that dictionaries with all nulls on rows won't match this branch anymore? Which I'm guessing is not the intent anyway. Makes sense.

Reply:
Also, dictionaries evaluated on a single row and dictionaries with indices being the same for all 'rows' won't be considered constant.

Reply:
Note to self: another major difference between design choices for implementing DecodedVector and peeling